client/lib/eventHubSender.ts

```diff
@@ -173,60 +174,59 @@ export class EventHubSender extends ClientEntity {
    * @param message The message to be sent to EventHub.
    * @return {Promise<rheaPromise.Delivery>} Promise<rheaPromise.Delivery>
    */
-  private _trySend(message: AmqpMessage, tag?: any, format?: number): Promise<rheaPromise.Delivery> {
+  private _trySend(message: rhea.Message, tag?: any, format?: number): Promise<rheaPromise.Delivery> {
     const sendEventPromise = new Promise<rheaPromise.Delivery>((resolve, reject) => {
       debug("[%s] Sender '%s', credit: %d available: %d", this._context.connectionId, this.name,
         this._sender.credit, this._sender.session.outgoing.available());
       if (this._sender.sendable()) {
         debug("[%s] Sender '%s', sending message: %O", this._context.connectionId, this.name, message);
-        let onRejected: Func<rheaPromise.Context, void>;
-        let onReleased: Func<rheaPromise.Context, void>;
-        let onModified: Func<rheaPromise.Context, void>;
-        let onAccepted: Func<rheaPromise.Context, void>;
+        let onRejected: Func<rheaPromise.EventContext, void>;
+        let onReleased: Func<rheaPromise.EventContext, void>;
+        let onModified: Func<rheaPromise.EventContext, void>;
+        let onAccepted: Func<rheaPromise.EventContext, void>;
         const removeListeners = (): void => {
-          this._sender.removeListener("rejected", onRejected);
-          this._sender.removeListener("accepted", onAccepted);
-          this._sender.removeListener("released", onReleased);
-          this._sender.removeListener("modified", onModified);
+          this._sender.removeAllListeners("rejected");
```

Comment:
This is the real change.....migrated from removeListener() to removeAllListeners()